src/gluonts/model/forecast.py

```diff
@@ -556,9 +556,12 @@ class OutputType(str, Enum):
 
 
 class Config(pydantic.BaseModel):
-    num_eval_samples: int = pydantic.Schema(..., alias="num_samples")
-    output_types: Set[OutputType]
-    quantiles: List[str]  # FIXME: validate list elements
+    num_eval_samples: int = pydantic.Schema(100, alias="num_samples")
+    output_types: Set[OutputType] = {"qunatiles", "mean"}
```

Comment:
Fix spelling of `qunatiles` to `quantiles`.

Reply:
Oh, thanks!